docs/plugins/index.rst

```diff
@@ -1,13 +1,40 @@
 Plugins
 =======
 
-Plugins add support for a type of content to Pulp. For example, the
-`file_plugin <https://github.com/pulp/pulp_file>`_ adds support for Pulp to manage files.
+Plugins add support for a type of content to Pulp. Pulp needs at least one plugin to manage
+content. For example, `file_plugin <https://github.com/pulp/pulp_file>`_ adds support for Pulp to
+manage files.
 
 All known Pulp plugins are listed below. If you are interested in creating a plugin, see
 :doc:`these docs <plugin-writer/index>`.
 
-And don't hesitate to :doc:`contact us <../troubleshooting/index>` with any questions during development.
+.. toctree::
+
+   plugin-api/overview
+   plugin-writer/index
+
+
+What plugins do
```

Comment:
This is one of three items in the Table of Contents that I'm just not sure if they should be left alone or capitalized like a title.  I'm leaning towards capitalizing them so that they stand out given that they are headings.